cmd/maya-agent/app/command/device-format.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
-//NewSubCmdFormatAndMount formats the specified disk
+//NewSubCmdFormat formats the specified disk
```

Comment:
Adding a space after `//` give a good readability in my opinion.

Reply:
Sure, addressed it in #667. PTAL.